src/Biogeochemistry.jl

```diff
@@ -92,20 +92,27 @@ struct NoBiogeochemistry <: AbstractBiogeochemistry end
 tracernames(tracers) = keys(tracers)
 tracernames(tracers::Tuple) = tracers
 
-@inline function has_biogeochemical_tracers(fields::NamedTuple, required_fields, grid)
-
-    for field_name in required_fields
-        if !(field_name in keys(fields))
-            fields = merge(fields, NamedTuple{(field_name, )}((CenterField(grid), )))
+add_tracer(tracers::Tuple, name, grid) = tuple(tracers..., name)
+add_tracer(tracers::NamedTuple, name, grid) = merge(tracers, NamedTuple{(name)}((CenterField(grid), )))
```

Comment:
```suggestion
add_tracer(tracers::NamedTuple, name, grid) = merge(tracers, NamedTuple(name => CenterField(grid)))
```

Reply:
Maybe `add_biogeochemical_tracer`? It's helpful if the names are more specific when searching the code for functions / names, etc.